.tests/integration/config.yaml

```diff
@@ -5,9 +5,9 @@ tools:
   common: "docker://marrip/common:1.1.0"
   default: "docker://gmsuppsala/somatic:develop"
 
- references:
-   genome: "/data/ref_genomes/hg19/bwa/BWA_0.7.10_refseq/hg19.with.mt.fasta"
+references:
+  genome: "/data/ref_genomes/hg19/bwa/BWA_0.7.10_refseq/hg19.with.mt.fasta"
```

Comment:
I was wondering where this file is located because it is not part of the repo, right?

Reply:
Its on Marvin at the location specified.

Reply:
ah, ok, is it to run the jenkins integration then?